scripts/f5-config.py

```diff
@@ -67,6 +67,7 @@
     r' rpc\r\n\r\n" }',
     r'create ltm monitor http /' + PART + '/' + PREFIX_NAME + '_MON_HTTP_NOVA_SPICE_CONSOLE {'
     r' defaults-from http destination *:6082 recv "200 OK" send "HEAD /spice_auto.html'
+    r' HTTP/1.1\r\nHost: rpc\r\n\r\n" }',
     r'create ltm monitor http /' + PART + '/' + PREFIX_NAME + '_MON_HTTP_NOVA_NOVNC_CONSOLE {'
     r' defaults-from http destination *:6082 recv "200 OK" send "HEAD /novnc_auto.html'
```

Comment:
This doesn't include the port change commit made here: https://github.com/rcbops/rpc-openstack/pull/1857/commits/27eb691e706b41755e4cb5cc416dab2cecffc3d6